RuleDocumentation/UseIdenticalMandatoryParametersForDSC.md

```diff
@@ -4,13 +4,24 @@
 
 ## Description
 
-The `Get-TargetResource`, `Test-TargetResource` and `Set-TargetResource` functions of DSC Resource must have the same mandatory parameters.
+For script based DSC resources the `Get-TargetResource`, `Test-TargetResource` and `Set-TargetResource` functions must have identical mandatory parameters that are also the keys in the corresponding `mof` file.
```

Comment:
I would remove the 'that are also the keys in the corresponding `mof` file.' While this is usually true, sometimes there are mandatory parameters that aren't keys.